sdk/storage/storage-file-share/test/directoryclient.spec.ts

```diff
@@ -630,10 +630,10 @@ describe("DirectoryClient", () => {
               ]
             },
             {
-              name: "Azure.Storage.File.DirectoryClient-createFile",
+              name: "Azure.Storage.File.ShareDirectoryClient-createFile",
               children: [
                 {
-                  name: "Azure.Storage.File.FileClient-create",
+                  name: "Azure.Storage.File.ShareClient-create",
```

Comment:
Something is off. FileClient -> ShareClient ?

Reply:
Fixed